capstone/src/main/resources/static/script.js

```diff
@@ -27,30 +27,97 @@ function getEvents() {
   });
 }
 
-/** Creates an event element. */
+/**
+ * Format event listing
+ */
 function createEventElement(event) {
   const eventElement = document.createElement('li');
   eventElement.className = 'event';
 
   // Name 
   const nameElement = document.createElement('p');
-  nameElement.innerText = event.text;
+  nameElement.innerText = event.name;
+  console.log(event.name);
+
+  const idElement = document.createElement('p');
+  idElement.innerText = event.datastoreId;
+  idElement.style.display = 'none';
 
+  // Review
+  const reviewElement = createReviewElement(event);
   /*
   Time
   Location
   Organization
   Description
   */
+  eventElement.appendChild(idElement);
   eventElement.appendChild(nameElement);
+  eventElement.appendChild(reviewElement);
   
   return eventElement;
 }
 
+/**
+ * Format review element and listing
+ */
+function createReviewElement(event) {
+  const reviewElement = document.createElement('span');
+
+  // Submission
+  const reviewTextElement = document.createElement('input');
+  reviewTextElement.class = 'review-text';
+  reviewTextElement.setAttribute('type', 'text');
+
+  // Future: option to add image
+
+  const reviewButtonElement = document.createElement('button');
+  reviewButtonElement.innerText = 'Submit Review';
+  reviewButtonElement.addEventListener('click', () => {
+    newReview(event.datastoreId, reviewTextElement.value);
+  });
+
+  // Container
+  const reviewContainer = document.createElement('div');
+  reviewContainer.innerText = 'Reviews:'
+  const reviews = event.reviews;
+
+  reviews.forEach((review) => {
+      const reviewTextElement = document.createElement('p');
```

Comment:
It's best to avoid redeclaring variables with the same name in different scopes (there is a `reviewTextElement` in the containing function, too) -- it can cause bugs due to confusion. Consider renaming one of them.